### PR TITLE
Roles and Profiles

### PR DESCRIPTION
**Pull request by joshsamuelson**

### CODE REVIEW

File: manifests/profile/copymod.pp [hunk 1]
```diff
@@ -1,4 +1,4 @@
-define bootstrap::copymod(
```
Comment (binford2k) [inline manifests/profile/copymod.pp]: defined types don't fit into roles & profiles. This should probably remain `bootstrap::copymod`

**Comment by binford2k** [comment]
This looks like a good first refactor. We'll likely refactor again into component classes or modules at sometime.

File: manifests/profile/copymod.pp (hunk 1 above)
Comment (joshsamuelson) [inline manifests/profile/copymod.pp]: Oops, I missed that one.  I don't think we're actually using this so I may just pull it altogether.

**Comment by joshsamuelson** [comment]
Agreed, this is going to take some iteration.  I also need to get the stuff from pltraining-lms since that's almost totally duplicated here.

**Pull request merged**